### PR TITLE
DAT-4370

### PR DESCRIPTION
**Pull request by rybmat**
## Links

* http://wikia-inc.atlassian.net/browse/DAT-4370
* more relevant links

## Description

hiding keyboard on transition between article and search page

## Reviewers

@Wikia/west-wing

### CODE REVIEW

**Comment by dianafa** [comment]
as I mentioned offline, we can think about moving `this.$('.side-search__input')` as a property and only referencing to it when focusing or bluring.

Rest LGTM

File: front/main/app/controllers/search.js [hunk 1]
```diff
@@ -5,33 +5,21 @@ const {Controller, computed, inject, $} = Ember;
...
 		},
 
 		onErrorPageClick() {
 			this.set('query', '');
-			this.focusSearchInput();
+			this.get('inputField').focus();
 		}
-	},
-
-	/**
```
Comment (SebastianMarzjan) [inline front/main/app/controllers/search.js]: I think this comment, while only regarding the action of caching the element, still holds true :)

**Comment by SebastianMarzjan** [comment]
One remark re: removed comment, rest LGTM

File: front/main/app/controllers/search.js (hunk 1 above)
Comment (rybmat) [inline front/main/app/controllers/search.js]: done

**Comment by SebastianMarzjan** [comment]
LGTM

**Pull request merged**